.github/workflows/e2e.yml

```diff
@@ -35,15 +36,16 @@ jobs:
       # resolution options, using npm release or a gh ref:
       #
       # option #1, set the version of soroban-js-client based on a npm release version
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 
+      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 1.0.0-beta.2
```

Comment:
wait this isn't out yet :sob:

Reply:
ok I released it so that this can pick it up when it finishes compiling :pray:

Reply:
oh, I'm an idiot and in a hurry :facepalm: didn't see the git ref override below